metaflow/plugins/conda/batch_bootstrap.py

```diff
@@ -60,16 +60,17 @@ def download_conda_packages(flow_name, env_id, datastore_type):
                     )
         else:
             # Import DATASTORES dynamically... otherwise, circular import
-            from metaflow.datastore import DATASTORES
+            from metaflow.plugins import DATASTORES
 
-            if datastore_type not in DATASTORES:
+            datastore_potentials = [d for d in DATASTORES if d == datastore_type]
```

Comment:
why datastore_potentials and not just datastore?

Reply:
or storage_impl?

Reply:
sure. whatever name you want :).